### PR TITLE
Updates youtube-to-mp3 from 3.9.9.54 to 3.9.9.57

### PR DESCRIPTION
**Pull request by kvorobyov**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

### CODE REVIEW

File: Casks/youtube-to-mp3.rb [hunk 1]
```diff
@@ -8,7 +8,9 @@
   homepage "https://www.mediahuman.com/youtube-to-mp3-converter/"
 
   livecheck do
-    skip "unversioned URL"
+    url :homepage
+    strategy :page_match
```
Comment (SMillerDev) [inline Casks/youtube-to-mp3.rb:12]: This it is the default.

**Comment by cho-m** [comment]
It would be better to create separate PRs for each Cask as it would be easier to merge.

Otherwise, we will need to fix the commit history before merge.

**Comment by kvorobyov** [comment]
> It would be better to create separate PRs for each Cask as it would be easier to merge.
looks like I fixed that, thanks.

**Review by cho-m — APPROVED** [review]
Thanks, @kvorobyov

**Pull request merged**